### PR TITLE
Add support for TMS commands

### PR DESCRIPTION
**Pull request by ArcaneNibble**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -848,6 +892,56 @@ impl MpsseCmdBuilder {
...
+    }
+
+    /// Clock TMS bits out while clocking TDO bits in.
+    ///
+    /// # Arguments
+    ///
+    /// * `mode` - TMS clocking mode.
+    /// * `data` - TMS bits.
+    /// * `tdi` - Value to place on TDI while clocking.
+    /// * `len` - Number of bits to clock out.
```
Comment (geomatsi) [inline src/lib.rs:931]: looks like a typo, should be 'to clock in'

Comment (geomatsi) [reply src/lib.rs:931]: Please ignore, my fault.

**Comment by geomatsi** [comment]
Hi @rqou 

Thanks for your contribution ! Looks good to me. Lets wait for feedback from Alex.

Regards,
Sergey

**Review by geomatsi — APPROVED** [review]

**Review by newAM — APPROVED** [review]
Looks good to me, thanks for the PR!

**Comment by newAM** [comment]
Released v0.1.1 with these changes: https://github.com/ftdi-rs/ftdi-mpsse/releases/tag/v0.1.1

**Pull request merged**